doc/mk_sdcard_image.sh

```diff
@@ -26,6 +31,7 @@ usage()
 	echo "  -s image-size   Optional, image size in GiB"
 	echo "  -u domain       Optional, unpack only specified domain: dom0, domd, domf, doma, domu"
 	echo "  -f              Optional, force rewrite of image file (useful for batch usage)"
+	echo "  -r              Optional, force repack of initiramfs and add DomA Image(from DomA dir) to it"
```

Comment:
Not sure we want this patch at all: this looks like a mixed functionality.
Even more, we do not ship the helper script with the builds.
at the end of the day, why DomA only?

Reply:
From my point of view: mk_sdk_image should provide the functionality to create a final image using build artifacts. For someone is very convenient to build DomA, and use product builds with modified DomA. As you know, due to Xen design we do not use boot.img in DomA, we use Image placed into Initramfs. Without this change, DomA integration is incomplete, despite Image is copied in to out artifacts.